guides/common/modules/proc_creating-a-content-filter-for-yum-content.adoc

```diff
@@ -1,8 +1,8 @@
-[id="Creating_a_Content_Filter_{context}"]
-= Creating a Content Filter
+[id="Creating_a_Content_Filter_for_YUM_Content_{context}"]
+= Creating a Content Filter for YUM Content
 
-Use this procedure to create a content filter.
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-content-filter_{context}[].
+Use this procedure to create a content filter for YUM content.
```

Comment:
```suggestion
Use this procedure to create a content filter for Yum content.
```
I think we chatted about this before. The capitalization gives the impression it is an acronym, which it isn't. The standard is `Yum`

Reply:
Yes, now I remember. Applied.